attestation-service/docs/policy.md

```diff
@@ -37,6 +37,17 @@ If the extension is registered in the EAR specification, the key will be positiv
 Otherwise, the key should be negative.
 The extension value can be any valid JSON, including nested objects.
 
+The default policy includes the extension `ear.trustee.identifiers`, which is used
+to store fields that identify the workload.
+With Confidential Containers these identifiers will usually be based on the Kata
+Agent Policy or other configuration fields in the InitData.
+Other runtimes might use other mechanisms, such as the event log.
+The `identifiers` extension includes two subfields.
+
+* `validated` identifiers are those that are bound to the hardware evidence via
```

Comment:
A question is that are there anything that is not bound to hardware evidence, but still can be captured by this policy?